mlflow/server/js/src/model-registry/components/ModelVersionView.js

```diff
@@ -44,13 +45,30 @@ export class ModelVersionViewImpl extends React.Component {
     isDeleteModalConfirmLoading: false,
     showDescriptionEditor: false,
     isTagsRequestPending: false,
+    isModelDeployed: false,
+    modelInferenceAPI: 'Not Deployed'
   };
 
   componentDidMount() {
     const pageTitle = `${this.props.modelName} v${this.props.modelVersion.version} - MLflow Model`;
     Utils.updatePageTitle(pageTitle);
   }
 
+  handleModelDeployment = () => {
```

Comment:
This function would call the deployment client. Need to check on access to model artifacts.